README.md

```diff
@@ -13,7 +13,9 @@ Source: [OpenFAST documentation](https://ganesh-openfast.readthedocs.io/en/lates
 
 ## Dependencies
 
-- [OpenFAST v3.5.0](https://openfast.readthedocs.io/en/main/source/install/index.html)
+- [OpenFAST v3.5.0](https://openfast.readthedocs.io/en/main/source/install/index.html), with the C++ API enabled
+  - Building from source: `cmake -DBUILD_OPENFAST_CPP_API=ON .. && make`
+  - Getting from Spack: `spack install openfast+cxx` (but build with an MPI compiler, see [issue](https://github.com/precice/openfast-adapter/issues/2))
 - [preCICE v3.0.0](https://precice.org/installation-overview.html) by installation from source from the [develop](https://github.com/precice/precice) branch
```

Comment:
```suggestion
- [preCICE v3.0.0](https://precice.org/installation-overview.html)
```
As preCICE v3 is packaged and released now, we may remove the link to the installation by source